config/locales/actions.en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  actions:
+    archive_prompt:
+      label: "Archive"
+      confirmation: "Are you sure you want to archive %{name}?"
```

Comment:
Hmm, my quest is to make this more readable and more identifiable what I am about to delete. Would it be worth it to have a conditiont so that if I am passing an arbitrary string or a date/int it displays with/out quotes? :)